crates/pop-contracts/src/lib.rs

```diff
@@ -4,6 +4,7 @@
 mod build;
 mod call;
 mod errors;
+mod init_tests;
```

Comment:
```suggestion
mod testing;
```
Reads better and is more future proof - we can place anything related to testing contracts within this module and present a better public api to the crate. All these exported functions at the root are not sustainable as the crate grows. A consumer can use `pop_contracts::testing::new_environment(..)` and `pop_contracts::testing::mock_build_process(..)`